com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceAmbientOcclusion.cs

```diff
@@ -148,12 +150,20 @@ private enum ShaderPasses
             internal ScreenSpaceAmbientOcclusionPass()
             {
                 m_CurrentSettings = new ScreenSpaceAmbientOcclusionSettings();
+                if (SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLES2)
+                    m_IsDeviceGLES2 = true;
```

Comment:
Given that GLES2 hardware will have a hard time running SSAO (same as PostFX), I wonder if it's useful to even go through the trouble of supporting it there...

Reply:
Will let @ellioman comment on that when he's back. I agree, but then we should warn nicely in the UI.